src/test/java/com/google/devtools/build/lib/runtime/BlazeOptionHandlerTest.java

```diff
@@ -297,6 +308,59 @@ public void testExpandConfigOptions_withConfig() throws Exception {
         .containsExactly("Found applicable config definition c0:config in file rc1: b");
   }
 
+  @Test
+  public void testExpandConfigOptions_withPlatformSpecificConfigEnabled() throws Exception {
```

Comment:
The new test methods are only subtly different, I find it difficult to understand what each of them tests.
Could you add short descriptions to explain what a test method does and how it's different from the related test methods? (E.g. one uses `--enable_platform_specific_config` therefore X, the other one uses `--config=platform_config` therefore Y.)

Reply:
Done.

Reply:
Thanks!